packages/hocs/withFocusTrap.jsx

```diff
@@ -1,27 +1,47 @@
 /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
-import React, { useRef } from 'react'
+import React, { useRef, useEffect } from 'react'
+import PropTypes from 'prop-types'
 
 // Returns an array of focusable elements in the order they are found in c
-const getFocusable = c =>
-  c.querySelectorAll('button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])')
+const selector =
+  'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"]), audio[controls], video[controls], [contenteditable]:not([contenteditable=false]'
```

Comment:
Can we remove `:not([tabindex="-1"])`. Example, we need to focus on a `tabIndex={-1}` in our [Footnote](https://github.com/telus/tds-core/blob/master/packages/TermsAndConditions/Footnote/Footnote.jsx#L268) component

Reply:
Although W3C says that the focus should be set to the first interact-able element (which in many cases is a "close" button). We need to look at what the user experience is. In the case of the TDS [Footnote](https://tds.telus.com/components/index.html#!/Footnote/1), we have a `tabIndex` of -1 on title, we want focus set there immediately so that the user knows where they are. Once focus has been moved away from the title, we do not want it keyboard focusable.

@cuginoAle What are your thoughts?

Reply:
@cuginoAle and I have decided that we will keep `[tabindex]:not([tabindex="-1"]` and allow for an opt-out of `withFocusTrap` setting initial focus.